src/app/public/modules/ag-grid/ag-grid.service.ts

```diff
@@ -204,7 +230,17 @@ export class SkyAgGridService {
 
   private dateFormatter(params: ValueFormatterParams, locale: string = 'en-us'): string | undefined {
     const dateConfig = { year: 'numeric', month: '2-digit', day: '2-digit' };
-    return params.value && params.value.toLocaleDateString(locale, dateConfig);
+    let date: Date = params.value;
+
+    if (date && typeof date === 'string') {
+      date = new Date(params.value);
+    }
+
+    let formattedDate = date && date.toLocaleDateString && date.toLocaleDateString(locale, dateConfig);
+
+    if (formattedDate !== 'Invalid Date') {
```

Comment:
How robust is this technique? Does the message remain `Invalid Date` if the browser language isn't English? I'd look for a more deterministic way of ensuring the date is valid.